requirements.txt

```diff
@@ -1,7 +1,8 @@
 wheel>=0.31.1
 numpy>=1.6.0
 scipy>=0.9.0
-boto3>=1.2.3
+botocore<=1.19
+boto3<=1.15
```

Comment:
It is probably better to keep a minimum version too.
You can have both inequalities with the syntax `boto3>=1.2.3, <=1.15`

Reply:
Ah, that makes sense. I'll keep the boto3 minimum version too. There wasn't a botocore inequality before, so I'll just leave that as is.